settings.php

```diff
@@ -248,6 +250,15 @@
         }
     }
 
+    // Admin_setting_check only exists in 4.5+, in lower versions fallback to a basic description.
+    if (class_exists('admin_setting_check')) {
+        $settings->add(new admin_setting_check('tool_objectfs/check_tokenexpiry', new token_expiry(), true));
+    } else {
+        $summary = (new token_expiry())->get_result()->get_summary();
+        $settings->add(new admin_setting_description('tool_objectfs/tokenexpirycheckresult',
+            get_string('checktoken_expiry', 'tool_objectfs'), $summary));
+    }
+
```

Comment:
Should this maybe be moved up further or under it's own heading? Right now for example with s3 set up it's at the bottom of Pre-Signed URLs Settings, which seems a little confusing, to an unknowing user it might look like it's referring to a pre-signed url token.

Reply:
yeah that is a really good point about confusing it for the pre signed url token. Have moved it to its own heading :+1: